crypto/evp/pmeth_lib.c

```diff
@@ -103,8 +103,17 @@ const EVP_PKEY_METHOD *EVP_PKEY_meth_find(int type)
 static EVP_PKEY_CTX *int_ctx_new(EVP_PKEY *pkey, ENGINE *e, int id)
 {
     EVP_PKEY_CTX *ret;
-    const EVP_PKEY_METHOD *pmeth;
+    const EVP_PKEY_METHOD *pmeth = NULL;
 
+    /*
+     * For new style use, the context is bound to the algo implementation
```

Comment:
"new style" won't make much sense in the future, when it isn't "new" any more.

Reply:
Good point.  What do you usually do when you want to say something like this?  (or should we go for "pink nirvana"?)

Reply:
How about "When using providers" or something similar?

Reply:
Thanks.  Will do

Reply:
Done